client/src/components/Sidebar/Sidebar.js

```diff
@@ -48,69 +48,80 @@ SidebarItem.defaultProps = {
 };
 
 /** A navigation list for the application */
-const Sidebar = ({ disabled, showUIPatterns }) => {
+const Sidebar = ({ disabled, showUIPatterns, loading }) => {
   let { path } = useRouteMatch();
   if (path.includes('accounts')) path = ROUTES.WORKBENCH;
 
   const unread = useSelector(state => state.notifications.list.unread);
+  const hideApps = useSelector(state => state.workbench.config.hideApps);
 
   return (
-    <Nav styleName="root" vertical>
-      <SidebarItem
-        to={`${path}${ROUTES.DASHBOARD}`}
-        label="Dashboard"
-        iconName="dashboard"
-        disabled={disabled}
-      />
-      <SidebarItem
-        to={`${path}${ROUTES.DATA}`}
-        label="Data Files"
-        iconName="folder"
-        disabled={disabled}
-      />
-      <SidebarItem
-        to={`${path}${ROUTES.APPLICATIONS}`}
-        label="Applications"
-        iconName="applications"
-        disabled={disabled}
-      />
-      <SidebarItem
-        to={`${path}${ROUTES.ALLOCATIONS}`}
-        label="Allocations"
-        iconName="allocations"
-        disabled={disabled}
-      />
-      <SidebarItem
-        to={`${path}${ROUTES.HISTORY}`}
-        label="History"
-        iconName="history"
-        disabled={disabled}
-      >
-        <HistoryBadge unread={unread} disabled={disabled} />
-      </SidebarItem>
-      {showUIPatterns && (
-        <SidebarItem
-          to={`${path}${ROUTES.UI}`}
-          label="UI Patterns"
-          iconName="copy"
-          disabled={disabled}
-        />
+    <>
+      {!loading && (
+        <Nav styleName="root" vertical>
```

Comment:
Based on [Wes's comment](https://github.com/TACC/Core-Portal/pull/447#issuecomment-887628458), I think we want to render the sidebar always, but hide the sidebar items if workbench loading is not complete.
```suggestion
    <Nav styleName="root" vertical>
      {!loading && (
        <>
```